src/Sulu/Bundle/MediaBundle/Resources/public/dist/components/area-selection/demo.html

```diff
@@ -0,0 +1,81 @@
+<!doctype html>
```

Comment:
Can be removed now, since I can test the overlay builtin the application now, right?